src/utils/viewer/viewer.cpp

```diff
@@ -321,10 +332,16 @@ Key Commands:
   esp::scene::SceneGraph* activeSceneGraph_ = nullptr;
   bool drawObjectBBs = false;
 
+  const std::string saved_transformations_directory = "saved_transformations/";
+  std::string cameraLoadTimeString_;
+  bool cameraSaved = false;
```

Comment:
Second, you may not need this boolean because
you can specify `lastCameraSave_` as a [Corrade::Containers::Optional ](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1Optional.html)